SPECS/iscsi-initiator-utils/iscsi-initiator-utils.spec

```diff
@@ -47,6 +45,8 @@ BuildRequires: flex bison doxygen kmod-devel systemd-units
 BuildRequires: autoconf automake libtool libmount-devel openssl-devel
 BuildRequires: isns-utils-devel
 BuildRequires: systemd-devel
+BuildRequires: systemd-rpm-macros
+BuildRequires: libselinux-devel
```

Comment:
I dont know about libselinux-devel but recently many packages faced issues with systemd-rpm-macros. This maybe a transient issue. Do you plan to revert the change later?

Reply:
i'm not sure.  i've asked on the channel about how we should workaround this pipeline/localbuild issue ... but haven't gotten any response.

i think the answer is that the solution for rpm-macros should be in another file and should not be added here.  but it is hard to merge something that i know doesn't build.

Reply:
removed workaround